### PR TITLE
PUBLIC_LAW - Add new STAFF and JUDICIAL roles configurations in RAS application

### PR DESCRIPTION
**Pull request by Karthikarenjith**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/AM-2462

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/roleconfig/role_publiclaw.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+          },
+          "primaryLocation": {
+            "mandatory": true
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "task-supervisor",
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_publiclaw.json:96]: please move case-allocator and task-supervisor to role_common.json

File: src/main/resources/roleconfig/role_common.json [hunk 2]
```diff
@@ -1691,7 +1692,8 @@
         },
         "attributes": {
           "jurisdiction": {
-            "mandatory": true
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:1696]: no need to add jurisdiction to special access cases. please remove "values": ["PUBLICLAW"]

File: src/main/resources/roleconfig/role_common.json [hunk 3]
```diff
@@ -1723,13 +1725,230 @@
         },
         "attributes": {
           "jurisdiction": {
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:1729]: no need to add jurisdiction to special access cases. please remove "values": ["PUBLICLAW"]

File: src/main/resources/roleconfig/role_common.json [hunk 4]
```diff
@@ -1659,7 +1659,8 @@
         },
         "attributes": {
           "jurisdiction": {
-            "mandatory": true
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:1663]: no need to add jurisdiction to special access cases. please remove "values": ["PUBLICLAW"]

File: src/main/resources/roleconfig/role_publiclaw.json [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+        "values": ["STANDARD"]
+      },
+      "classification": {
+        "mandatory": true,
+        "values": ["PUBLIC", "PRIVATE"]
+      },
+      "attributes": {
+        "jurisdiction": {
+          "mandatory": true,
+          "values": ["PUBLICLAW"]
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_publiclaw.json:25]: remove this as its already in role_common.json

**Review by sankar-padakula — DISMISSED** [review]

File: src/main/resources/roleconfig/role_common.json [hunk 6]
```diff
@@ -2112,9 +2141,9 @@
     ]
   },
   {
-    "name": "hmcts-ctsc",
-    "label": "HMCTS CTSC",
-    "description": "HMCTS CTSC role for ctsc users",
+    "name": "hearing-manager",
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:2144]: hearing manager for CTSC appearing two times. one for publiclaw and another for privatelaw. can you please merge as one

File: src/main/resources/roleconfig/role_common.json [hunk 7]
```diff
@@ -2126,14 +2155,47 @@
...
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["PUBLICLAW"]
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "hearing-viewer",
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:2174]: hearing-viewer for CTSC appearing two times. one for publiclaw and another for privatelaw. can you please merge as one

File: src/main/resources/roleconfig/role_common.json [hunk 8]
```diff
@@ -2195,13 +2286,173 @@
...
           },
           "primaryLocation": {
             "mandatory": true
           }
         }
       }
     ]
+  },
+  {
+    "name": "task-supervisor",
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:2299]: task-supervisor for CTSC appearing two times. can you please merge as one

File: src/main/resources/roleconfig/role_common.json [hunk 9]
```diff
@@ -2195,13 +2286,173 @@
...
+          "jurisdiction": {
+            "mandatory": true,
+            "values": ["PUBLICLAW", "PRIVATELAW"]
+          }
+        }
+      }
+    ]
+  },
+  {
+    "name": "case-allocator",
```
Comment (sankar-padakula) [inline src/main/resources/roleconfig/role_common.json:2329]: case-allocator for CTSC appearing two times. can you please merge as one

**Review by sankar-padakula — APPROVED** [review]

**Review by sankar-padakula — APPROVED** [review]

**Review by nitinsahu1 — APPROVED** [review]

**Pull request merged**